salt/tests/unit/modules/test_cordon.py

```diff
@@ -0,0 +1,123 @@
+from parameterized import parameterized
+
+import salt
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+
+import metalk8s_cordon
+
+
+class CordonTestCase(TestCase, LoaderModuleMockMixin):
```

Comment:
Rename it with metalk8s to be consistent

```suggestion
class Metalk8sCordonTestCase(TestCase, LoaderModuleMockMixin):
```